pandas/tests/indexes/ranges/test_range.py

```diff
@@ -533,3 +535,24 @@ def test_isin_range(self, base):
         result = base.isin(values)
         expected = np.array([True, False])
         tm.assert_numpy_array_equal(result, expected)
+
+    @pytest.mark.parametrize("copy", [True, False])
+    @pytest.mark.parametrize("name", [None, "foo"])
+    @pytest.mark.parametrize("ordered", [True, False])
+    def test_astype_category(self, copy, name, ordered, simple_index):
+        super().test_astype_category(copy, name, ordered, simple_index)
```

Comment:
hmm we don't usually call the super functions as they are *already* called so this is confusing. i think you can remove. (assume my assertion is true)